src/mmw/js/src/modeling/views.js

```diff
@@ -805,7 +805,10 @@ var ResultsView = Marionette.LayoutView.extend({
                 if (err && err.timeout) {
                   tmvModel.setTimeoutError();
                 } else {
-                  tmvModel.setError(err);
+                    var message = err.error === 'NO_LAND_COVER' ?
```

Comment:
Very large areas of interest should time out, which should result in this: https://github.com/kellyi/model-my-watershed/blob/f80396fcb52d7f421421d7060db179cc0ac44aa9/src/mmw/js/src/core/models.js#L225-L231. In case that is not happening, maybe because Spark Job Server times out before the front-end polling (unlikely but possible?), perhaps we should return a more descriptive error.